tests/README.md

```
@@ -0,0 +1,213 @@
+# Kiwi 测试目录
+
+本目录包含 Kiwi 项目的各类测试用例，参考 [kiwi-cpp](https://github.com/OpenAtomFoundation/kiwi) 项目的测试组织结构。
+
+## 📁 目录结构
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)